webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/PurgeDeletedInventories.scala

```diff
@@ -85,8 +85,18 @@ class PurgeDeletedInventories(
             val error = Chained(s"Error when purging deleted nodes inventories older than ${TTL} days", err)
             ScheduledJobLoggerPure.error(error.fullMsg)
         })
+        .unit
       import zio.Duration.fromScala as zduration
-      ZioRuntime.unsafeRun(prog.delay(zduration(updateInterval)).repeat(Schedule.spaced(zduration(updateInterval))).forkDaemon)
+      // Effect should be running forever and every defect caught into the UIO
+      ZioRuntime.unsafeRun(
+        prog
+          .catchAllDefect(err =>
+            ScheduledJobLoggerPure.error(s"Error in process or purging deleted nodes inventories older than ${TTL} days", err)
```

Comment:
If we go there, the fiber died. I believe it's the one with the "unit of work for that iteration", ie just one purge delete, and thus the next schedule will just happen. Is that so ?